### PR TITLE
Detect and help resolve import problems in js2model.

### PR DESCRIPTION
**Pull request by charlesmchen**
PTAL @floatplane

### CODE REVIEW

File: src/js2model [hunk 1]
```diff
@@ -20,6 +20,42 @@
...
+except ImportError:
+    print
+    print 'Could not import "jsonschema" library.'
+    print 'For installation instructions, see: https://github.com/ahupp/python-magic'
+    print 'ie.'
+    print 'brew install python'
+    print 'pip install jsonschema'
+    print
+    sys.exit(1)
+
```
Comment (floatplane) [inline src/js2model]: how's about 
```python
import importlib
import sys

def safe_import(module):
    try:
        return importlib.import_module(module) 
    except ImportError:
        print
        print 'Could not import "' + module + '" library.'
        print 'For installation instructions, see: https://github.com/ahupp/python-magic'
        print 'ie.'
        print 'brew install python'
        print 'pip install ' + module
        print
        sys.exit(1)

safe_import("jsonref")
safe_import("mako")
safe_import("jsonschema")
safe_import("not_a_real_module")
```

output:
```bash
$ python
Python 2.7.10 (default, Jan 27 2016, 15:15:04) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import importlib
>>> import sys
>>> 
>>> def safe_import(module):
...     try:
...         return importlib.import_module(module) 
...     except ImportError:
...         print
...         print 'Could not import "' + module + '" library.'
...         print 'For installation instructions, see: https://github.com/ahupp/python-magic'
...         print 'ie.'
...         print 'brew install python'
...         print 'pip install ' + module
...         print
...         sys.exit(1)
... 
>>> safe_import("jsonref")
<module 'jsonref' from '/usr/local/lib/python2.7/site-packages/jsonref.pyc'>
>>> safe_import("mako")
<module 'mako' from '/usr/local/lib/python2.7/site-packages/mako/__init__.pyc'>
>>> safe_import("jsonschema")
<module 'jsonschema' from '/usr/local/lib/python2.7/site-packages/jsonschema/__init__.pyc'>
>>> safe_import("not_a_real_module")

Could not import "not_a_real_module" library.
For installation instructions, see: https://github.com/ahupp/python-magic
ie.
brew install python
pip install not_a_real_module
```

**Comment by floatplane** [comment]
we should figure out what's up with the tests failing before merging. looks like something in the setup may have changed - they don't get run that often, so maybe Travis changed something on their end. I'll TAL now.
 
Otherwise LGTM

**Comment by floatplane** [comment]
The build error sounds transient, if https://github.com/travis-ci/travis-ci/issues/5461 is to be believed. So I wouldn't wait for that to pass.

File: src/js2model (hunk 1 above)
Comment (charlesmchen) [inline src/js2model]: Beautiful, done.

**Pull request merged**